Add APICAST_ACCESS_LOG_BUFFER option (#1248)
3scale/APIcast: doc/parameters.md

Defines the file that will store the access logs.

### APICAST_ACCESS_LOG_BUFFER

**Values:** integer
**Default**: nil

> Allows access log writes to be writes in chunks on bytes, so less system calls

Reviewer: ```suggestion
Allows access log writes to be included in chunks of bytes, resulting on fewer system calls
```

> that improves the performance of the whole gateway.

Reviewer: ```suggestion
that improve the performance of the whole gateway.
```
